.vscode/launch.json

```diff
@@ -0,0 +1,28 @@
+{
+  "configurations": [
```

Comment:
Is this supposed to be checked in here?

Reply:
No they're not. I removed them and added that path to gitignore.